bench/axis_opencode_native_fallback.go

```
@@ -0,0 +1,254 @@
+		return fmt.Errorf("native role request models = %v (all requests %v), want [general explore]", f.roles, f.models)
+	}
+	if strings.Join(f.responses, ",") != "ROLE_RESPONSE general,ROLE_RESPONSE explore" {
+		return fmt.Errorf("native role responses = %v", f.responses)
+	}
+	return nil
+}
+
+func nativeFallbackRequestText(request nativeFallbackRequest) string {
+	var text strings.Builder
+	for _, message := range request.Messages {
+		text.WriteString(fmt.Sprint(message.Content))
```

golangci-lint (2.12.2)

[error] QF1012: Use fmt.Fprint(...) instead of WriteString(fmt.Sprint(...))

(staticcheck)